sdk/trace/build.gradle

```diff
@@ -12,6 +12,7 @@ ext.propertiesDir = "build/generated/properties/io/opentelemetry/sdk/trace"
 
 dependencies {
     api project(':api:all'),
+            project(':api:semconv'),
```

Comment:
maybe just `semconv`? Not under `api`?

Reply:
I wasn't really sure which was better. Looking at the specs, it looks like semantic conventions are described under a section called "data", so I agree that it probably doesn't make sense to have it under API here, either.  👍